src/constants.ts

```diff
@@ -11,15 +11,19 @@ export const REPOS = {
 
 export const NUM_SUPPORTED_VERSIONS = 4;
 
+export const FOUR_DAYS_AGO = 4 * 24 * 60 * 60 * 1000;
```

Comment:
Can we make this `3.9` days just to handle the couple of minutes of lag / drift that could occur around the cron job.